microf/__main__.py

```diff
@@ -209,6 +209,12 @@ def accept(self, filename):
             new_md['well_letter'] = old_md['well_letter']
             new_md['well_nr'] = int(old_md['well_nr'])
             new_md['site'] = int(old_md['site'])
+            # If the IC6K filename contains a Z position, use it
+            if old_md['z_pos']:
```

Comment:
Would be better to have:

    if old_md['z_pos'] is not None:

otherwise the `if` branch would still be skipped if Z-pos is present but 0.

Reply:
Z-pos 0 should never happen, but the statement is clearer, thus I added it as well.